xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-ui/src/XWiki/Lightbox/LightboxAdministration.xml

```diff
@@ -0,0 +1,332 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<xwikidoc version="1.4" reference="XWiki.Lightbox.LightboxAdministration" locale="">
+  <web>XWiki.Lightbox</web>
+  <name>LightboxAdministration</name>
+  <language/>
+  <defaultLanguage>en</defaultLanguage>
+  <translation>0</translation>
+  <creator>XWiki.Admin</creator>
+  <creationDate>1639138843000</creationDate>
+  <parent>WebHome</parent>
+  <author>XWiki.Admin</author>
+  <contentAuthor>XWiki.Admin</contentAuthor>
+  <date>1640076427000</date>
+  <contentUpdateDate>1639244117000</contentUpdateDate>
+  <version>5.6</version>
```

Comment:
`mvn xar:format`  should reset the version to `1.1` normally, this is weird that it isn't.

Reply:
Similarly the dates should be removed. Are you sure you ran it? :)

Reply:
I did ran it. But I looked closely now and I did messed up the folder structure. Fixing it..
Sorry for the trouble